packages/remote-replay-launcher/src/url.utils.ts

```
@@ -1,4 +1,30 @@
-export function getPort(url: URL): number {
+export function extractHostnameAndPort(urlString: string): {
+  hostname: string;
+  port: number;
+} {
+  let url: URL;
+  try {
+    url = new URL(urlString);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  } catch (_error) {
+    throw new Error(`Invalid app URL: '${urlString}'`);
+  }
+
+  if (!["http:", "https:", "ws:", "wss:", "ftp:"].includes(url.protocol)) {
```

Does 'https://' work? Should we throw an error if they try using 'https'?